server/app/controllers/DisplayableMessage.java

```diff
@@ -0,0 +1,39 @@
+package controllers;
+
+import views.components.ToastMessage.ToastType;
+
+public class DisplayableMessage {
+
+  private final String message;
+  private final Severity severity;
+
+  public DisplayableMessage(String message, Severity severity) {
+    this.message = message;
+    this.severity = severity;
+  }
+
+  public String getMessage() {
+    return message;
+  }
+
+  public Severity getSeverity() {
```

Comment:
Is there a strong reason to create a distinct enum type for this or could we use `ToastType` instead?